ISISComputingGroup/ibex_gui
# Ticket2539:  Switch to new NICOS protocol

### Description of work

Modified the GUI to communicate with NICOS using ZeroMQ.

### To test

* Get the new NICOS core from \\olympic\Babylon5\Scratch\Dom Oram\nicos-core
* Run bin\nicos-daemon (I had some difficulty merging this into our nicos repo)
* Confirm that on the NICOS scripting screen this connects
* Confirm you can send a script on the NICOS screen
* Confirm that pointing to an instrument that isn't running NICOS gives a sensible message and periodically attempts to reconnect
* Confirm unit tests run

---

#### Code Review

    - Manual system tests for the functionality should be added if there are no automated tests
    - Manual system tests can be removed from the template if they are covered by suitable automated tests

### Functional Tests


### Final Steps
